modules/camera/camera_factory.py

```diff
@@ -30,6 +29,6 @@ def create_camera(
         case CameraOption.OPENCV:
             return camera_opencv.CameraOpenCV.create(width, height)
```

Comment:
Add in a config for the OpenCV camera as well, even if it's empty. This way it's setup and easy to modify in the future

Reply:
In `camera_opencv.py` just make create take in a config Object as well, and then write a comment `#TODO: apply camera configs to camera here` to indicate it will be done sometime in the future